.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Proposal'
+labels: proposal
+assignees: reynoldsalec,pirog
+---
+
+Good day fine Tandemite! It would appear you have an idea to improve our company. Bravo! Please fill out the following issue template so that we can get your idea fully fleshed out and ready for lift off!
+
+## Feeling the Pain
+
+Is this idea designed to assuage an existing pain or solve an existing problem you see at the company? Describe those pain points here. Have you already had discussions with your colleagues that they'd be happy with you sharing? The more people we can show as feeling the pain, the clearer the case is to act.
+
+## Missing the Boat
+
+Is this idea designed to take advantage of an opportunity that Tandem is currently missing out on? Describe the opportunity here.
+
+## What is the Status Quo?
+
+What are we currently doing to address this _category_ of issue? What strengths do the current methods have and where do they fall short? We should acknowledge our current efforts and explicitly decide if this idea would augment or replace them.
+
+## Proposed Solution(s)
+
+This is where the rubber meets the road. We've identified the problem, what are you thinking of for solutions? These don't have to be perfect, and its okay if they even sound a little crazy. Its also okay if you have more than one solution in mind that you'd like to evaluate. The point here is to start the ideas flowing so that we can diverge before working down to a solution we'd like to implement.
+
+## Costs
+
+Every change we make has a potential cost. Do your best to outline the costs you can see from this proposal from your current vantage point. While humans are terrible at estimating, there should be at least a good amount of thought put into this.
+
+## Benefits
+
+What value do you see this brining to the organization? Can you quantify, even in broad strokes what the potential upside is for Tandem as a whole? Does this have direct sales potential? Is the proposal about increasing effeciency? How many hours do you think we can save per week? How many projects do you think this could save a quarter? This is a *hard* question to accurately answer, and its okay if you aren't sure or your estimate is off, the point is to think about this and give your best estimate on what benefits we can expect. Here are some potential categories of benefits:
```

Comment:
> How many projects do you think this could save a quarter?

I don't get this one; does it mean how many projects would this idea apply to a quarter?

Reply:
I meant essentially, in the case that a project is doomed, as in, we're going to completely blow it because we keep doing `x`, and we screw up at least `y` projects per `time frame`, so if we do this, we will not screw up on those `y` projects

Reply:
k; I don't think that kind of question belongs here; if we are at that point with a  project this process will be too slow for it and a direct intervention is needed on that project.  If we have more than one project in that state in a quarter ... well then we prolly don't have a company to improve.

Reply:
Another example might be adding certain pre-launch SEO checks. This doesn't necessarily save hours, but it might save our bacon with a client 🥓

Reply:
In reading this, I feel @serundeputy is talking about the specific `How many` question, and @dustinleblanc is talking about the entire section.

Either way, are the questions below each heading meant to be answered literally? or are they simply ideas for detail that could go in there? I was thinking the latter, which might change the concern here.

I agree though, the "how many" questions would be difficult to answer meaningfully in many cases. So many variables.

Reply:
Right these are example questions, they are meant to be a jumping off point to spark ideas about different benefits you might see from an initiative. I'd love proposals on other questions to ask or other details to add in here / substitute.